qa/pull-tester/rpc-tests.py

```diff
@@ -38,112 +38,113 @@
     # Scripts that are run by the travis build process
     # Longest test should go first, to favor running tests in parallel
     # vv Tests less than 5m vv
-    'wallet.py',
```

Comment:
The PR currently conflicts with `master` on these changes. Given that (as I understand it) the order here is irrelevant to how the CI runner being introduced here will run the tests, I suggest undoing this reordering so there is no difference in the list between this PR and master. It would simplify the PR review and make it less prone to conflict.

Reply:
Ah yes, looks like the tests moved again. @yasserisa Should revert my organization to the list and just use what is currently on master. Once the list is sharded while running the organization won't matter, unless someone is sitting there timing each one to see which comment block it should be in.

Reply:
The timing blocks are mostly irrelevant; they were introduced by upstream Bitcoin Core, but don't really apply to us as our test resource utilisations are different.